src/room_member.cpp

```diff
@@ -66,6 +66,35 @@ void RoomMember::HandleWifiPackets(const RakNet::Packet* packet) {
     }
 }
 
+
+void RoomMember::HandleRoomInformationPacket(const RakNet::Packet* packet) {
+    RakNet::BitStream stream(packet->data, packet->length, false);
+
+    // Ignore the first byte, which is the message id.
+    stream.IgnoreBytes(sizeof(RakNet::MessageID));
+
+    RakNet::RakString room_name;
+    stream.Read(room_name);
+    room_information.name.assign(room_name.C_String(), room_name.GetLength());
+    stream.Read(room_information.member_slots);
```

Comment:
Is member_slots a fixed-width integer? if not, it should be.
Do not ever have platform-dependent code (such as the width of int / unsigned int) in network transmissions